src/template_project/analysis/model_template.py

```diff
@@ -4,20 +4,12 @@
 from statsmodels.iolib.smpickle import load_pickle
 
 
-def fit_logit_model(data, data_info, model_type):
+def fit_logit_model(data, formula, model_type):
     """Fit a logit model to data.
 
     Args:
         data (pandas.DataFrame): The data set.
-        data_info (dict): Information on data set stored in data_info_template.yaml. The
-            following keys can be accessed:
-            - 'outcome': Name of dependent variable column in data
-            - 'outcome_numerical': Name to be given to the numerical version of outcome
-            - 'columns_to_drop': Names of columns that are dropped in data cleaning step
-            - 'categorical_columns': Names of columns that are converted to categorical
-            - 'column_rename_mapping': Old and new names of columns to be renamend,
-                stored in a dictionary with design: {'old_name': 'new_name'}
-            - 'url': URL to data set
+        formula (str): The formula for the logit model.
         model_type (str): What model to build for the linear relationship of the logit
             model. Currently implemented:
             - 'linear': Numerical covariates enter the regression linearly, and
```

Comment:
```suggestion
            - 'linear_index': Numerical covariates enter the Logit index linearly;
```
This is confusing otherwise, as linear would usually refer to the overall regression, which is a Logit in this case.